### PR TITLE
added auto restart where it was missing. And did the same for logging

### PR DESCRIPTION
**Pull request by LBONTEN**
Added restart: always in the main docker-compose file. For dev environments this can be overridden.
Also added logging: *default-logging 
The label "logging=true" was added to user-facing services like resources and search.

### CODE REVIEW

**Review by nvdk — CHANGES_REQUESTED** [review]
As said in the chat, couple of changes required.
Please enable logging on cache, dispatcher and identifier. other changes look good to me!

File: docker-compose.yml [hunk 1]
```diff
@@ -157,18 +166,18 @@ services:
     volumes:
       - ./data/files/consumer-files/besluiten:/consumer-files/
     restart: always
-    labels:
-      - "logging=true"
     logging: *default-logging
   search:
     image: semtech/mu-search:0.9.0-beta.7
```
Comment (nvdk) [inline docker-compose.yml:171]: this one is still missing the logging label

Comment (LBONTEN) [reply docker-compose.yml:171]: Fixed! That should do it

Comment (LBONTEN) [reply docker-compose.yml:171]: That should do it! :)

**Review by nvdk — APPROVED** [review]

**Pull request merged**